msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
+rem
+rem       1. Files installed to `bin\` folder use Directory='BinFolder' and
+rem          all others use the pattern Directory='INSTALLFOLDER:\path\'.
+rem
+rem       2. The `git.exe` file in `BinFolder` is given the explicit
+rem          Id='GetExe' all other files are anonymous (do not have an Id).
+
+rem Build the .msi.
+rem
+rem TODO: Correctly set the bind paths (change the -b gitfileshere switch)
+rem       to the real source location for the binaries to include in the
+rem       installation package.
+rem
+rem TODO: Set the output paths appropriately.
+rem
+wix\candle.exe GitProduct.wxs GitComponents.wxs GitCredStoreComponents.wxs GitPoshComponents.wxs -o build\obj\  -ext WixUtilExtension
+wix\light.exe build\obj\GitProduct.wixobj build\obj\GitComponents.wixobj build\obj\GitCredStoreComponents.wixobj build\obj\GitPoshComponents.wixobj -o build\msi\git.msi -ext WixUtilExtension -b gitfileshere -b ..\installer -sval
+
+rem Build the bundle.
```

Comment:
I wouldn't do that. The bundle works around a lot of the issues that can occur when distributing naked MSI packages for the internet. For example, the Windows Installer will register the location the MSI is launched from as the source. If launched straight from the browser that location could be some location deep in the browser cache. If the MSI requires a repair (for example, the `post-install.bat` missing because it deletes itself) then the prompt will be for the MSI in the browser cache.

Additionally, it may make sense to create separate MSI for the posh-git and git-credentials and the bundle creates a single unified experience for installing all of those.

The bundle works around many small issues distributing naked MSIs over the internet and adds many useful features.

Instead, I recommend shipping the bundle as was done in gitsetup.exe. In WiX v4 we'll finish the feature to create an admin image (i.e. extracts the MSIs embedded in the bundle) for those users that want to deploy the naked MSI(s) via GPO. For those users, the bundle is the single distribution container (useful if multiple MSIs are used for Git for Windows).